benchmarks/time/src/main/scala/strawman/collection/immutable/HashSetBenchmark.scala

```diff
@@ -136,6 +136,15 @@ class HashSetBenchmark {
   }
 
   @Benchmark
+  def transform_zip(bh: Blackhole): Unit = bh.consume(xs.zip(xs))
+
+  @Benchmark
+  def transform_zipWithIndex(bh: Blackhole): Unit = bh.consume(xs.zipWithIndex)
+
+  @Benchmark
+  def transform_lazyZip(bh: Blackhole): Unit = bh.consume(xs.lazyZip(xs).map((_, _)))
+
+  @Benchmark
   def transform_unzip(bh: Blackhole): Unit = bh.consume(zipped.unzip)
```

Comment:
is there any recommendation for when (not) to add new benchmarks to the project? These are going to add several hours to the running time

Reply:
No recommendation so far. New benchmarks are always welcomed for now. In my case anyway I only run a subset of the benchmarks when I run them.

Reply:
Also, what I do sometimes is that I group related benchmarks together so that it’s simpler to run just the benchmarks that I’m interested in: https://github.com/scala/collection-strawman/blob/master/benchmarks/time/src/main/scala/strawman/collection/GroupMapBenchmarks.scala